_docs/technical/2017-05-15-wallet-backend.md

```diff
@@ -69,7 +69,7 @@ The wallet web API is enabled by default. It uses the `IP:PORT` specified with a
 `--wallet-address` option (defaults to: `127.0.0.1:8090`).
 
 Documentation for wallet web API is available
-[here](https://cardanodocs.com/technical/wallet/api/).
+[here](/technical/wallet/api/v0).
```

Comment:
Is that V0 or V1? The doc is about a subset of the V1 API, right?

Reply:
This is indeed a link to V0, the old and (so far) current one. Since we now also host V1 documentation, we had to change a bit the paths to the documentations which now includes the version number. Hence, I had to update an existing link pointing to V0.

There's no link to V1 documentation however, as far as I understand, we will communicate on that on "private" channels until official announcement.

Reply:
> There's no link to V1 documentation however, as far as I understand, we will communicate on that on "private" channels until official announcement.

@KtorZ I think I know what you mean, but let me try to paraphrase it for @jcmincke 's benefit: in the _README_ we are concealing this link, but it will still be _exposed_ to the public, provided one know the link a-priori. Correct? I think JC concern here is just making sure by accident we are forgetting to ship the documentation for V1 😁

Reply:
Correct. The API will be accessible provided one knows the link or is simply smart enough to turn 'v0' into 'v1' in the URL. 
The rational here isn't to have anything secret but simply not yet advertised (or at least, that is what I understood from our strategy here ^.^)

The line above just make sure nothing breaks in the existing behaviour by still providing a valid link to the old API where it was already done :+1: